apps/labs/posts/pep-517-build-system-popularity.md

```diff
@@ -0,0 +1,615 @@
+  <tr style={{ background: '#eef' }}><th colspan='2'>flit</th><th></th></tr>
+  <tr style={{ background: '#eef' }}><td></td><td>`flit_core.buildapi`</td><td align='right'>276</td></tr>
+  <tr style={{ background: '#eef' }}><td></td><td>`flit.buildapi`</td><td align='right'>4</td></tr>
+  <tr style={{ background: '#eef' }}><td></td><td>`flit_scm:buildapi`</td><td align='right'>3</td></tr>
+  <tr style={{ background: '#eef' }}><td></td><td>(custom)</td><td align='right'>1</td></tr>
+  <tr style={{ background: '#eef' }}><td></td><td>`flit_gettext.scm`</td><td align='right'>1</td></tr>
+  <tr style={{ background: '#efe' }}><th colspan='2'>pdm</th><th></th></tr>
+  <tr style={{ background: '#efe' }}><td></td><td>`pdm.backend`</td><td align='right'>37</td></tr>
+  <tr style={{ background: '#efe' }}><td></td><td>`pdm.pep517.api`</td><td align='right'>4</td></tr>
+  <tr style={{ background: '#efe' }}><td></td><td>`pdm.backend.intree`</td><td align='right'>1</td></tr>
+  <tr style={{ background: '#fee' }}><th colspan='2'>scikit-build-core</th><th></th></tr>
+  <tr style={{ background: '#fee' }}><td></td><td>`scikit_build_core.build`</td><td align='right'>28</td></tr>
+  <tr style={{ background: '#fee' }}><td></td><td>(custom)</td><td align='right'>2</td></tr>
+</table>
+
+</div>
+
+<div style={{textAlign: 'center'}}>
+
+<figure id="fig-1" style={{width: 'auto', margin: '0 2em', display: 'inline-block', verticalAlign: 'top'}}>
+  <img src="/posts/pep-517-build-system-popularity/fig1.svg"
+    alt="Pie chart of build backend use. Setuptools is used by 78% of packages,
+    poetry 8.4%, hatchling 6.5%, flit 3.8%. The other backends are lumped together
+    to form 2.6%."
+  />
+
+  <figcaption>Figure 1. Cumulative use of build backends</figcaption>
+</figure>
+
+<figure id="fig-2" style={{width: 'auto', margin: '0 2em', display: 'inline-block', verticalAlign: 'top'}}>
+  <img src="/posts/pep-517-build-system-popularity/fig2.svg"
+    alt="Pie chart of setuptools backend use. 56% packages do not declare
+    build-backend, 22% use setuptools.build_meta backend. Other backends
+    represent less than 1%. All three pieces of the pie are of equal length
+    as the setuptools piece in figure 1, leaving the remainder open."
+  />
+
+  <figcaption>Figure 2. Use of setuptools backends</figcaption>
+</figure>
+
+</div>
+
+Setuptools was used as a build system for almost 79%
+of the tested packages. Seven out of ten packages using setuptools do not
+declare a build backend in `pyproject.toml` — they rely on the tools
+running `setup.py` when no backend is declared.
+
+The three next top ranking build system families are Poetry (8.4% packages),
+Hatchling (6.5% packages) and Flit (3.8%). Build systems other than the four
+already mentioned amount for 2.6% of packages. They include both generic Python build
+systems (such as `pdm-backend`), as well as more specialized tools such as:
```

Comment:
I'm not sure the split is generic vs. specialized - pdm and the backends other than setuptools in the sentence before are all for pure Python code which Maturin/scikit-build-core/meson-python are the main options (next to setuptools) for packages with compiled code. Perhaps better to phrase it like that?